### DISCUSSION TITLE
Copilot4Eclipse 1.3.2 (20241111) Available

### DISCUSSION THREAD

**Discussion by wayneparrott** [Genuitec/Copilot4Eclipse-community#50] [Announcements]
Copilot4Eclipse 1.3.2 has been released (20241111).

This update provides key fixes and improvements as outlined in the  [Changelog](https://www.genuitec.com/products/copilot4eclipse/docs/resources/changelog).

Install from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/copilot4eclipse).

**Comment by will-lauer** [comment]
Reinstalling from Eclipse Marketplace today (20241116) installed version 1.3.0,202410241454 not 1.3.2 even though the Marketplace description listed 1.3.2. Similarly, the update site https://www.genuitec.com/updates/copilot4eclipse still seems to show 1.3.0, not 1.3.2.

**Comment by wayneparrott** [comment]
Hmm! That's odd. 1.3.2 is the only active version for download. I just confirmed 1.3.2 installs by default with a new eclipse install, new workspace. 

I'm wondering if there may be a CDN update issue. Do you mind sharing the general region you are installing from? I'm in Dallas, TX. Also can you try uninstalling and do a clean?

W

**Reply by will-lauer** [reply]
I'm installing in central Illinois, but I get the same results when VPN'd into work in Buffalo, NY.

This is odd. Looks like a fresh install is picking up the correct version of the plugins, but I couldn't seem up update an older installation of the same version of eclipse. However, after installing the fresh eclipse, I CAN update the old eclipse. The best guess I have is that they are sharing a bundle pool and somehow that is overriding something, causing the new vesrion to not download.
